src/nifreeze/utils/iterators.py

```diff
@@ -114,10 +125,10 @@ def bvalue_iterator(size: int | None = None, **kwargs) -> Iterator[int]:
     bvalues : :obj:`list`
         List of b-values corresponding to all orientations of the dataset.
```

Comment:
Same comment as above: noting this is an optional/kwarg, and documenting the behavior when it is not provided. There is other examples of the latter, so if the behavior is the same, a simple C&P would work.

Reply:
```suggestion
        List of b-values corresponding to all orientations of the dataset.
        Please note that ``bvalues`` is a keyword argument and MUST be provided
        to generate the volume sequence.
```